sci-biology/emboss/emboss-6.6.0.ebuild

```diff
@@ -1,28 +1,18 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2015 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Header: $
 
-EAPI="4"
+EAPI=5
 
-inherit emboss eutils
-
-#EBO_PATCH="4"
-#EBOV=${PV/_p*}
+inherit autotools-utils emboss eutils
 
 EBO_PATCH=""
 EBOV=${PV}
 
 DESCRIPTION="The European Molecular Biology Open Software Suite - A sequence analysis package"
-SRC_URI="
```

Comment:
Homepage is missing:

https://travis-ci.org/gentoo-science/sci/jobs/48324066:
> HOMEPAGE.missing 1
sci-biology/embassy/embassy-6.6.0.ebuild